StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/SavedPaymentMethods/SavedPaymentMethodsAddPaymentMethodViewController.swift

```diff
@@ -18,13 +18,7 @@ class SavedPaymentMethodsAddPaymentMethodViewController: UIViewController {
     // MARK: - Read-only Properties
     weak var delegate: SavedPaymentMethodsAddPaymentMethodViewControllerDelegate?
     lazy var paymentMethodTypes: [PaymentSheet.PaymentMethodType] = {
-        guard let intent = intent else {
-            return [.card]
-        }
-        let paymentMethodTypes = PaymentSheet.PaymentMethodType.recommendedPaymentMethodTypes(from: intent)
-            .filter { $0 == .card }
-        assert(!paymentMethodTypes.isEmpty, "At least one payment method type must be available.")
-        return paymentMethodTypes
+        return [.card]
```

Comment:
nit: This doesn't need to be a lazy var anymore

Reply:
done!

Reply:
sorry, should've been more specific, you can replace this with a simple let 

`let paymentMethodTypes = [.PaymentSheet.PaymentMethodType.card]`